### ISSUE TITLE
Use this repo's issue list also for potential Trainer Meeting topics

### ISSUE THREAD

**Issue by brownsarahm** [carpentries/trainers#180]
## Summary

Add a form template type and use issues in this repository to collect issues for trainer meetings. 

## Background

We currently have topics that "could be good meeting topics" spread out across at least: Slack, Curriculum, and the discussions on this repo. 

Examples: 
- [using the rubric in demos](https://github.com/carpentries/instructor-training/pull/1360#issuecomment-1109930829)

## Implementation

- create the issue template using the below starter
- create a label, trainermeeting 
- notify trainers

### Pros

Keeps content public, in one place.  Keeps all activity by trainer leadership (who has the power to schedule trainer meetings) in once place.  Creates more access for trainer community to suggest topics, as is their power. 

### Cons

Could be too intimidating for trainers who are not fluent in GitHub. We could create a Google form that goes to Core Team to complete this form though. 

## Possible Alternatives

- keep it ad-hoc
- use an etherpad 
- Google form

 ## Appendices

This will render as a form:
```
name: Template for Trainer Meeting Topics
description: Suggest a topic for a trainer meeting
title: "[Trainer Meeting]:  a short topic "
labels: ["trainermeeting"]
body:
  - type: markdown
    attributes:
      value: |
       Thanks for contributing a topic!
  - type: textarea
    id: intro
    attributes:
      label:Introduce the topic
      description: describe how the topic would be introduced for the trainer meeting including and background
    validations:
      required: true
  - type: checkboxes
    id: time
    attributes:
      label: Which trainer meeting time would you host?
      description: you may select both or none to say that you do not want to host the topic
      options:
        - Meeting 1 (UTC 14:00)
        - Meeting 2 (UTC 22:00
  - type: textarea
    id: scheduling
    attributes:
      label: Are there any scheduling concerns? 
      description: this may include timeliness of the topic or limits on your ability to host the meeting
    validations:
      required: true
```

**Comment by karenword** [comment]
This looks great! I wonder if we should offer an option to schedule an "other" meeting time, as this was requested by some Trainers in the meetings last week. It gets a little complicated with zoom room reservations and calendar invites, so we'd want to be an exception not the norm, but we could probably make something work.

**Comment by karenword** [comment]
It would also be desirable to allow people to sign up to host a meeting without proposing a topic. So maybe this could be a "meeting host signup and/or topic proposal" form?

**Comment by jcoliver** [comment]
I like this proposal

**Comment by marklcrowe** [comment]
I like this proposal

**Comment by brownsarahm** [comment]
I like this proposal and I figured out how to preview the [form](https://github.com/brownsarahm/trainers/blob/previewform/.github/ISSUE_TEMPLATE/trainer_meeting_topic.yml)

**Comment by brownsarahm** [comment]
closed via #192
